recipe/package_data.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/setup.py b/setup.py
+index 6021604..1ac1cdd 100644
+--- a/setup.py
++++ b/setup.py
+@@ -43,6 +43,21 @@ problem before tyring to use the epics package.
+ """
+ 
+ 
++# determine which libca / libCom dll is appropriate
++import platform
++mach = platform.machine()
++if mach.startswith('arm'):
++    nbits = 'arm'
++else:
++    nbits = platform.architecture()[0].replace('bit', '')
```

Comment:
For future us, this is the output of the command ([docs](https://docs.python.org/3/library/platform.html#platform.architecture)):
```py
In [1]: import platform

In [2]: platform.architecture()
Out[2]: ('64bit', 'ELF')
```
so, the first element is taken, and the `bit` string is stripped from that.
:+1: